sensor_msgs/PointCloud2 pcd_in
---
int64 n_clusters
